### ISSUE TITLE
vertex creation fails silently if field value is not quoted properly

### ISSUE THREAD

**Issue by nagarajasr** [orientechnologies/orientdb#5937]
OrientDB 2.2.0-SNAPSHOT develop@r76eb89664869753ee518b741fd2009d2255877c8

i made a typo while trying to create a vertex using console. the field "id" value quote is not closed. the command silently completes without throwing any error but no record is created:

```sql
orientdb {db=testdb}> create vertex c1 set id="13

orientdb {db=testdb}> select from c1


0 item(s) found. Query executed in 0.003 sec(s).
orientdb {db=testdb}> desc c1

CLASS 'C1'

Records..............: 0
Super classes........: [V]
Default cluster......: c1 (id=17)
Supported clusters...: c1(17), c1_1(18), c1_2(19), c1_3(20)
Cluster selection....: round-robin
Oversize.............: 0.0

```

**Comment by nagarajasr** [comment]
i have the same issue with the insert command too:

```
orientdb {db=testdb}> insert into c1 set id="23

orientdb {db=testdb}> select from c1


0 item(s) found. Query executed in 0.002 sec(s).
orientdb {db=testdb}> 
```

**Comment by luigidellaquila** [comment]
Hi @nagarajasr 

I'm on it, the fix will be available in short

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @nagarajasr 

I just pushed a fix on both 2.1.x and develop branches

Thanks

Luigi
